ggu/main.c

```diff
@@ -422,21 +422,6 @@ process_single_packet(struct rte_mbuf *pkt, struct ggu_config *ggu_conf)
 		 */
 		ip6hdr = l3_hdr;
 
-		/*
-		 * XXX #63 Given that IPv6 ntuple filter doesn't check
-		 * the destination address, it must be done here.
-		 * If the IPv6 packet is not destined to
-		 * the Gatekeeper server, redirect the packet properly.
-		 */
-		if (hw_filter_ntuple_available(back) &&
-				memcmp(ip6hdr->dst_addr,
-					back->ip6_addr.s6_addr,
-					sizeof(ip6hdr->dst_addr)) != 0) {
```

Comment:
Doesn't it "solve" issue #63? If so, why are we adding more references to that issue instead of having this patch to close it?